scripts/release/rpm/mariner.dockerfile

```diff
@@ -21,8 +21,8 @@ COPY . .
 RUN dos2unix ./scripts/release/rpm/azure-cli.spec && \
     REPO_PATH=$(pwd) CLI_VERSION=$cli_version PYTHON_PACKAGE=python3 PYTHON_CMD=python3 \
     rpmbuild -v -bb --clean scripts/release/rpm/azure-cli.spec && \
-    cp /usr/src/mariner/RPMS/*/azure-cli-${cli_version}-1.*.rpm /azure-cli-dev.rpm && \
-    mkdir /out && cp /usr/src/mariner/RPMS/*/azure-cli-${cli_version}-1.*.rpm /out/
+    cp /usr/src/*/RPMS/*/azure-cli-${cli_version}-1.*.rpm /azure-cli-dev.rpm && \
```

Comment:
What is the new path on Azure Linux 3.0?

Reply:
`/usr/src/azl/RPMS/x86_64/azure-cli-2.62.0-1.azl3.x86_64.rpm`

Reply:
Let's add some example names in the comment.